### ISSUE TITLE
Tripplite Srcoolnet12K

### ISSUE THREAD

**Issue by Bowl2much** [librenms/librenms#10979]
Hi All,
I have a Tripplite SRCOOLNET12K which is apparently is very similar to a WEBSNMPCARD.  It is detected as Tripplite and shows networking graphs, but it does not show any of the environmental graphs. Specially it does not show Temperature, Humidity, Fan, Etc.
I am completely new to LibreNMS and not much better with SNMP so I am nowhere near skilled enough to try to fix this myself.  I will attach the relevant info.

- [x] Please provide **ALL** info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x] Please provide as much detail as possible.
- [x] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.
https://p.libren.ms/view/745b09e6
https://p.libren.ms/view/0693cde3
https://p.libren.ms/view/ffd87e89

Thank you for any and all help.
I am learning fast and I am liking LibreNMS the more I use it.

**Comment by PipoCanaja** [comment]
Hi @Bowl2much 
Please ensure that the snmpwalk data does not contain any CarriageReturns. The one you attached currently is not usable because of those lines split in 2. 
Thanx

**Comment by PipoCanaja** [comment]
By the way, there is already a PR ( #10239 ) ongoing on this family of devices. They seem to have a buggy SNMP implementation. You should probably follow the evolutions, and test the patch as it gets ready for it.

**Comment by Bowl2much** [comment]
I saw those weird carriage returns. But I copied that text verbatim from the web interface of LibreNMS.
So now have run a snmpwalk with instructions found somewhere on the documentation:
https://p.libren.ms/view/46f120ff

I saw that PR previously.  But I dont know enough about what they are talking about, and it looks like its broke.  
And that PR certainly is yet another slightly different card.  
If nothing else this info that I add may help with completeness and I can help with testing and make the whole thing better.

**Comment by PipoCanaja** [comment]
Still some CR around I would say. Did you do it completely from CLI ? snmpwalk ...  | pbin... ?

**Comment by Bowl2much** [comment]
Sorry for the delay.
I did it again for completeness:
https://p.libren.ms/view/a1eac427
the command I used was: 
'snmpwalk -v2c -c community ipaddress'

Some of those lines are really long and I think the pastebin might be messing with them.  I dont know how to do them any better.  This is completely from the command line.  I ran the command in Putty, hightlighted, copied and pasted in the pastebin.  Nothing else.

But I truly do appreciate your help.
